### PR TITLE
ARM CI

### PR DESCRIPTION
**Pull request by tiagoshibata**
Build WinML for ARM in CI builds

### CODE REVIEW

**Comment by snnn** [comment]
The "Download python" and "Run python installer" tasks can be deleted and replaced by 

```
  - task: UsePythonVersion@0
    inputs: 
      versionSpec: '3.7' 
      addToPath: true 
      architecture: x64   #x64 or x86
```

File: tools/ci_build/github/azure-pipelines/win-arm-crosscompile-ci-pipeline.yml [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-        arguments: '/S /NoRegistry=1 /AddToPath=0 /RegisterPython=0 /D=$(Build.BinariesDirectory)\packages\python'
-      timeoutInMinutes: 10
+        versionSpec: '3.x'
     - task: BatchScript@1
       displayName: 'setup env'
       inputs:
         filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\setup_env.bat'
         modifyEnvironment: true
         workingFolder: '$(Build.BinariesDirectory)'
-    - task: CmdLine@1
```
Comment (tiagoshibata) [inline tools/ci_build/github/azure-pipelines/win-arm-crosscompile-ci-pipeline.yml]: The CMake version hosted in onnxruntime is too old and doesn't support VS 2019, so we're using the system provided one

Comment (orilevari) [reply tools/ci_build/github/azure-pipelines/win-arm-crosscompile-ci-pipeline.yml]: On other pipelines that are using the cmake_generator flag, the cmake_path flag is simply omitted. Are you sure we need to explicitly set the path to the system cmake?

Comment (tiagoshibata) [reply tools/ci_build/github/azure-pipelines/win-arm-crosscompile-ci-pipeline.yml]: Probably not if it's in PATH, thanks for the hint

**Pull request merged**